.github/workflows/push.yml

```diff
@@ -31,3 +31,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: seL4/ci-actions/style@master
+      with:
```

Comment:
not clear on syntax, but when does one use a hyphen?

Reply:
The hyphen is for lists, so `steps` is a list of dicts, and this dict has two entries `uses` and `with`, where `with` is another dict.